website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md

```diff
@@ -28,6 +28,6 @@ The final base-recalibrated BAM was converted to CRAM using Samtools view and va
 
 ### Variant calling
 
-Prior to variant calling, the variant calling interval list was split to enable parallelization. Using the GATK PrintReads tool, the WellformedReadFilter was applied to reads. Variant calling was then applied to reads that passed the filter using GATK (v3.5) HaplotypeCaller with the parameters --max_alternate_alleles 3 (sufficient for human data),  --ERC GVCF, and --read_filter OverclippedRead (to reduce false positives resulting from contamination). The resulting GVCFs were merged using Picard MergeVcfs and the final VCF file was validated using GATK ValidateVariants. Variant metrics were calculated using Picard CollectVariantCallingMetrics.
+Prior to variant calling, the variant calling interval list was split to enable parallelization. Using the GATK PrintReads tool, the WellformedReadFilter was applied to reads. Variant calling was then applied to reads that passed the filter using GATK (v3.5) HaplotypeCaller with the parameters --max_alternate_alleles 3 (sufficient for human data),  --ERC GVCF, and --read_filter OverclippedRead (to reduce false positives resulting from contamination). The resulting GVCFs were merged using Picard MergeVcfs and then reblocked using GATK ReblockGVCF. The final reblocked GVCF file was validated using GATK ValidateVariants. Variant metrics were calculated using Picard CollectVariantCallingMetrics.
```

Comment:
I wonder if the GQ bands should be mentioned here like they are in the exome methods section?

Reply:
The default WGS pipeline doesn't specify them in HaplotypeCaller (it uses GATK3.5), but we can include the reblocked GQ bands: -GQB 20 -GQB 30 -GQB 40

Reply:
Updated this.